prov/efa/src/rdm/rxr_pkt_entry.h

```diff
@@ -216,27 +215,21 @@ struct rxr_pkt_entry {
 	struct rxr_pkt_entry *next;
 
 	/** @brief information of send buffer */
-	struct rxr_pkt_sendv send;
+	struct rxr_pkt_sendv *send;
 
 	/** @brief Work request struct used by rdma-core */
-	union {
-		struct efa_send_wr send_wr;
-		struct efa_recv_wr recv_wr;
-	};
+	struct efa_send_wr *send_wr;
 
 	/** @brief buffer that contains data that is going over wire */
-	char *wiredata;
+	char wiredata[0];
```

Comment:
Still missing a comment or static assert saying that `sizeof(rxr_pkt_entry)` should be 64 bytes

Also need to add a pad for debug mode so that `sizeof(rxr_pkt_entry)` is 128 bytes

Reply:
With this change `sizeof(rxr_pkt_entry)` is not 64 bytes. (it is 80)

There will follow up commits to make `sizeof(rxr_pkt_entry)` . Once that commits is in, I will add the `static_assert`.

I will add some todo item.

Reply:
OK. We can't merge until you change this to 64 bytes, right? If it's currently not 64 bytes, how are you able to tell that the performance is good?

Reply:
This PR already improve performance, and is also a sizeable change. so I think we should merge it. Meanwhile, I am active working on bring done the header size.